lib/conjur/api/variables.rb

```diff
@@ -105,6 +105,17 @@ def variable_values(varlist)
       end
     end
 
+    # Fetch all visible variables that expire within the given
+    # interval (expressed as an ISO8601 duration). If no duration is
+    # specifed, all variables that are set to expire will be returned.
+    #
+    # param [String] duration ISO8601 duration 
```

Comment:
How is it ambiguous? (Or is it actually the `(now...(now+duration))` interval? The doc doesn't make it clear (it mentions interval but doesn't say when it starts). If so, it should support something like ActiveSupport's `2.days` -- which means `#to_i` would work great.)